saleor/core/context_processors.py

```diff
@@ -19,8 +19,10 @@ def default_currency(request):
 
 # request is a required parameter
 # pylint: disable=W0613
-def categories(request):
-    return {'categories': Category.tree.root_nodes()}
+def navigation(request):
+    menus = Menu.objects.prefetch_related(
+        'items', 'items__collection', 'items__category', 'items__page').all()
+    return {settings.NAVIGATION_CONTEXT_NAME: list(menus)}
```

Comment:
Since this is not really a configuration (I don't think anyone would ever need to change it) it could be moved to a constant somewhere in this module.